docs-website/sidebars.js

```diff
@@ -334,11 +353,11 @@ module.exports = {
         },
         {
           "Operator Guides": [
-            {
-              type: "doc",
-              id: "docs/managed-datahub/operator-guide/setting-up-remote-ingestion-executor",
-              className: "saasOnly",
-            },
+            // {
```

Comment:
commenting out for now - ideally this will have a URL re-direct to `/docs/managed-datahub/remote-executor/about` ; once we confirm that's working, we'll remove the file & this commented out section